From 14290be9159214675c381734c2175afaf5d632e2 Mon Sep 17 00:00:00 2001
From: Geliang Tang <geliang.tang@suse.com>
Date: Mon, 28 Nov 2022 08:30:35 +0800
Subject: [PATCH] Revert "Linux 5.10.156"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit 6d46ef50b123f2da3871690e619f5169eb97af92.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 166f87bdc190..8ccf902b3609 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 156
+SUBLEVEL = 155
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.3

